cliquet/resource.py

```diff
@@ -299,12 +299,12 @@ def collection_get(self):
         """Collection ``GET`` endpoint: retrieve multiple records.
 
         :raises: :exc:`~pyramid:pyramid.httpexceptions.HTTPNotModified` if
-            ``If-Modified-Since`` header is provided and collection not
+            ``If-Match`` header is provided and collection not
```

Comment:
It is the reverse modification, ``If-Modified-Since`` is ``If-None-Match`` and ``If-Unmodified-Since`` is ``If-Match``